ci/travis/install-toolchains.sh

```diff
@@ -32,6 +32,20 @@ install_toolchains() {
     install /dev/stdin "${target}"
     7z x -bsp0 -bso0 "${target}" -o"${targetdir}"
     rm -f -- "${target}"
+    (
+      # Add Clang/LLVM binaries to somewhere that's in already PATH
+      # (don't change PATH itself, to avoid invalidating Bazel's cache or having to manage environment variables)
```

Comment:
if we don't do this, would this invalidate Bazel's cache every time? or this is just a one time cost

Reply:
It's a fixed cost, but might happen more than once per build, depending on how many different PATHs we call Bazel with (for testing, wheels, etc.)

That's secondary to the concern of not having to reload `PATH` everywhere though. It's so much easier if we don't even have to worry about managing `PATH` everywhere.

Reply:
can't we just `export PATH=...` in .bashrc?

Reply:
I just thought it'd be better to not modify `.bashrc` and `PATH` everywhere unless I really need to. Note that this is Windows-only. But I can look into doing that instead.